pkg/installer/installation/prerequisites.go

```diff
@@ -90,8 +90,18 @@ func determineOS(ctx *util.Context) (string, error) {
 	return osID, err
 }
 
+const hostnameCommand = `
+fqdn=$(hostname -d)
```

Comment:
`hostname -d` returns the dns domainname and not the fqdn. this seems to be incorrect here.

Reply:
That said, I think it is fair to assume a usable short hostname if no long hostname can be concluded.
I would still avoid the less portable `==`. Something like this:
```
fqdn=$(hostname -f)
[ "$fqdn" = localhost ] && fqdn=$(hostname)
echo "$fqdn"
```